cohorts/utils_api.py

```diff
@@ -161,17 +161,19 @@ def get_cohort_job(request, filters, data_version, cohort_info):
     return_level = request.GET['return_level']
     select = levels[return_level]
 
-    # Get the SQL
-    if request.GET['return_sql'] in [True, 'True']:
-        cohort_info['cohort']['sql'] = get_bq_string(
-            filters=filters, fields=select, data_version=data_version,
-            order_by=select[-1:])
-
     if request.GET['return_level'] != "None":
+        # Get the SQL
+        if request.GET['return_sql'] in [True, 'True']:
+            cohort_info['cohort']['sql'] = "\t({})\n\tUNION ALL\n".format(get_bq_string(
+                filters=filters, fields=select, data_version=data_version,
```

Comment:
This produces a query that ends with UNION ALL, which doesn't seem to make sense?

Reply:
Removed UNION ALLs